src/test/java/ru/spbstu/reader/VcfReaderTest.java

```diff
@@ -0,0 +1,63 @@
+package ru.spbstu.reader;
+
+import org.junit.jupiter.api.Test;
+import ru.spbstu.reader.dto.VcfRecord;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class VcfReaderTest {
```

Comment:
JUnit5 test classes and methods should have default package visibility [see](https://rules.sonarsource.com/java/RSPEC-5786)